Makefile

```diff
@@ -16,24 +16,18 @@ TEMPLATE_PREVIEW_API_KEY ?= "my-secret-key"
 DOCKER_CONTAINER_PREFIX = ${USER}-${BUILD_TAG}
 
 NOTIFY_CREDENTIALS ?= ~/.notify-credentials
-CF_MANIFEST_FILE ?= manifest-${CF_SPACE}.yml
 
 NOTIFY_APP_NAME ?= notify-template-preview
 CF_APP = notify-template-preview
 
-CODEDEPLOY_PREFIX ?= notifications-template-preview
-
 CF_API ?= api.cloud.service.gov.uk
 CF_ORG ?= govuk-notify
-CF_HOME ?= ${HOME}
-$(eval export CF_HOME)
 CF_SPACE ?= development
 
 DOCKER_IMAGE = govuknotify/notifications-template-preview
 DOCKER_IMAGE_TAG = ${DEPLOY_BUILD_NUMBER}
 
 DOCKER_IMAGE_NAME = ${DOCKER_IMAGE}:${DOCKER_IMAGE_TAG}
-DOCKER_TTY ?= $(if ${JENKINS_HOME},,t)
```

Comment:
Removing this has the effect of not attaching a tty to the container when running locally, which means you can't stop the the app running in Docker with ctrl-c anymore. Not sure of there's a simple way of getting that behaviour back, because it's quite handy being able to stop the container with ctrl-c locally

Reply:
good spot! thanks